docs/Installation.rst

```diff
@@ -53,8 +55,9 @@ On Ubuntu 14.10, one will need to run the following script to satisfy all depend
 
 .. sourcecode:: bash
 
-    sudo apt-get install build-essential fpc postgresql postgresql-client \
-         gettext python2.7 iso-codes shared-mime-info stl-manual cgroup-lite
+    sudo apt-get install build-essential openjdk-8-jre openjdk-8-jdk fpc \
```

Comment:
It requires an external PPA, which is often frowned upon... I think it's best not to require external PPAs in order to install CMS...